lib/features/children/generosity_challenge/models/task.dart

```diff
@@ -12,21 +12,18 @@ class Task extends Equatable {
     required this.description,
     required this.onTap,
     required this.buttonText,
-    required this.feedbackImage,
```

Comment:
I would suggest to keep feedback per day logic since it's already implemented and can be useful for upcoming challenges, even if we do not use it for the current one (I assume bc of simplification).